File: jimmer-example/spring-boot-with-jimmer-multiple-datasource/src/main/java/org/jimmer/jimmerdatasouce/domain/College.java
package org.jimmer.jimmerdatasouce.domain;

import org.babyfish.jimmer.sql.*;

import java.util.List;

/**
 * @author jhlz
 * @version x.x.x
 */
@Entity
@Table(name = "db_college")
public interface College {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    long id();

    @Key
    String collegeName();

    @OneToMany(mappedBy = "college")
    List<Student> students();
}

File: jimmer-example/spring-boot-with-jimmer-kt/src/main/kotlin/org/jimmer/domain/College.kt
package org.jimmer.domain

import org.babyfish.jimmer.sql.*

/**
 * @author jhlz
 * @version x.x.x
 */
@Entity
@Table(name = "db_college")
interface College {

    @Id
    @GeneratedValue(strategy =  GenerationType.IDENTITY)
    val id: Long

    @Key
    val collegeName: String

    @OneToMany(mappedBy = "college")
    val students: List<Student>
}